### PR TITLE
Alabama

### PR DESCRIPTION
**Pull request by erinxocon**
Roll tide

### CODE REVIEW

**Comment by erinxocon** [comment]
image

Some of the things seem to be missing, but I think they are missing on others so it's ok? Let me know what needs to be fixed!

File: states/alabama/elections.toml [hunk 1]
```diff
@@ -3,7 +3,60 @@ date = 2020-07-14
 name = "Congressional Primary Run-Off (if needed)"
 original_date = 2020-07-14
 
+[20200714.registration]
+received_by = 2020-06-29
```
Comment (tannewt) [inline states/alabama/elections.toml:9]: Looks like you can register online and hand deliver the registration forms as well. ( https://www.alabamainteractive.org/sos/voter_registration/voterRegistrationWelcome.action ) Therefore `online_by` and `in_person_by` should be listed as well.

Comment (tannewt) [reply states/alabama/elections.toml:9]: Interesting that the primary notes different dates.

Screen Shot 2020-06-27 at 10 16 23 AM

File: states/alabama/elections.toml [hunk 2]
```diff
@@ -3,7 +3,60 @@ date = 2020-07-14
...
+sources = ["https://www.sos.alabama.gov/sites/default/files/2020%20Voter%20Guide%205-15-2020.pdf",
+           "https://www.alabamainteractive.org/sos/voter_registration/voterRegistrationWelcome.action"]
+
+[20200714.poll]
+in_person_by = 2020-07-14T19:00:00
```
Comment (tannewt) [inline states/alabama/elections.toml:15]: Can add `in_person_start` as well since polls open at 7am.

File: states/alabama/elections.toml [hunk 3]
```diff
@@ -3,7 +3,60 @@ date = 2020-07-14
...
+           "https://www.sos.alabama.gov/alabama-votes"]
+
+[20200714.absentee]
+postmark_by = 2020-07-13
+in_person_by = 2020-07-13T17:00:00
+sources = ["https://www.sos.alabama.gov/sites/default/files/2020%20Voter%20Guide%205-15-2020.pdf", 
+           "https://www.sos.alabama.gov/alabama-votes/voter/absentee-voting"]
+
+[20200714.absentee.application]
+received_by = 2020-07-09
```
Comment (tannewt) [inline states/alabama/elections.toml:26]: `in_person_by` here too

**Review by tannewt — CHANGES_REQUESTED** [review]
A couple notes on the 7/14 election that also applies to November. Thanks!

File: states/alabama/elections.toml (hunk 2 above)
Comment (erinxocon) [reply states/alabama/elections.toml:15]: This is grand! I was wondering if there was a field for that since I was able to find the start and end times!

**Comment by erinxocon** [comment]
@tannewt Fixed and pushed. Apologies if you were the one to resolve the conversations!

**Review by tannewt — APPROVED** [review]
Looks great! Thank you!

**Pull request merged**